### PR TITLE
chore(i18n): crowdin sync

### PR DESCRIPTION
**Pull request by ovflowd**

### CODE REVIEW

File: pages/be/download/releases.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
 layout: download-releases.hbs
 title: Папярэднія рэлізы
-modules: '<код>NODE_MODULE_VERSION</код> адносіцца да нумара версіі ABI (двайковы інтэрфейс праграмы) Node.js, які выкарыстоўваецца для вызначэння таго, якія версіі двайковых файлаў C++ дапаўненняў, скампіляваных Node.js, можна загружаць без паўторнай кампіляцыі. Раней, у больш ранніх версіях, яно захоўвалася як шаснаццатковае значэнне, але цяпер прадстаўлена ў выглядзе цэлага ліку.'
```
Comment (rodion-arr) [inline pages/be/download/releases.md:4]: Looks like `<code>` HTML tag was translated, shouldn't be like that

Comment (ovflowd) [reply pages/be/download/releases.md:4]: Indeed. Feel free to update this PR with the adjustment

Comment (rodion-arr) [reply pages/be/download/releases.md:4]: done, resolving

**Review by rodion-arr — APPROVED** [review]

**Review by aymen94 — APPROVED** [review]

**Review by AugustinMauroy — APPROVED** [review]

**Comment by AugustinMauroy** [comment]
strange [this workflows](https://github.com/nodejs/nodejs.org/actions/runs/6208213537/job/16854158626?pr=5782) doesn't work

**Comment by mikeesto** [comment]
@AugustinMauroy I think this was fixed in https://github.com/nodejs/nodejs.org/pull/5794 . Going to merge this now.

**Comment by mikeesto** [comment]
Hmm not sure what happened here. `npx prettier --write` doesn't appear to have worked in CI, I had to run it manually

**Pull request merged**